Source/Microsoft.Teams.Apps.DLLookup/ClientApp/src/apis/api-list.ts

```diff
@@ -77,8 +77,9 @@ export const createUserPageSizeChoice = async (payload: {}): Promise<AxiosRespon
 }
 
 export const getAuthenticationMetadata = async (windowLocationOriginDomain: string, loginHint: string): Promise<AxiosResponse<string>> => {
-    let url = `${baseAxiosUrl}/authenticationMetadata/GetAuthenticationUrlWithConfiguration?windowLocationOriginDomain=${windowLocationOriginDomain}&loginhint=${loginHint}`;
-    return await axios.get(url, undefined, false);
+    const payload = { windowLocationOriginDomain: encodeURIComponent(windowLocationOriginDomain), loginhint: loginHint };
```

Comment:
Wouldn't this now be double-encoded?

Reply:
Yes it's getting double encoded at server as we are using _HttpUtility.UrlEncode_ at server, so removed encoding from here